### ISSUE TITLE
[Bug] zettlr is not opening web link when clicking on the tooltip link

### ISSUE THREAD

**Issue by schlichtanders** [Zettlr/Zettlr#2354]
### Description

When creating a simple weblink, clicking on the link on the black toolbar popup
![image](https://user-images.githubusercontent.com/4255480/130627178-61e17e24-1e26-45cd-9757-228a3d77c7c5.png)

 does not work but raises the following error popup
![image](https://user-images.githubusercontent.com/4255480/130626812-d20cf41e-05d8-46f4-9301-523e1bccfb9a.png)

### Reproducing

1. install zettlr via current nixos 21.05
2. create a new document
3. create a new hyperlink to some web-site
4. hover over this hyperlink
5. click the link in the appearing toolbox (looks black for me) 

### Zettlr Version

Stable (most recent version)

### Specify version

1.8.9

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Nixos 21.05.1226.f77036342e2 (Okapi)

**Comment by schlichtanders** [comment]
starting zettlr from the command line gives me this extra information
```
(node:2) ExtensionLoadWarning: Warnings loading extension at /home/ssahm/.config/Zettlr/extensions/nhdogjmejiglipccpnnnanhbledajbpd:
Unrecognized manifest key 'browser_action'.
Unrecognized manifest key 'update_url'.
Permission 'contextMenus' is unknown or URL pattern is malformed.
Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
```
maybe that helps

**Comment by tfmoraes** [comment]
@schlichtanders I'm using Zettlr in NixOS-Unstable and Gnome and this is working. If I start from command line I see the same informations:

```
❯ zettlr --trace-warnings                                                                                                                                         teste 
(node:2) ExtensionLoadWarning: Warnings loading extension at /home/thiago/.config/Zettlr/extensions/nhdogjmejiglipccpnnnanhbledajbpd:
  Unrecognized manifest key 'browser_action'.
  Unrecognized manifest key 'update_url'.
  Permission 'contextMenus' is unknown or URL pattern is malformed.
  Cannot load extension with file or directory name _metadata. Filenames starting with "_" are reserved for use by the system.
```

Maybe you need to activate something in you configuration.nix. See my configurations https://github.com/tfmoraes/nixoscfg

**Comment by schlichtanders** [comment]
thank you very much. I am using KDE, which may be the reason.

I also tried marktext as an alternativ appimage + electron based markdown tool and the links don't work their either with the identical error. Typora on the other hand works, however this is not using appimage for building, maybe hence the difference.

It is really good to know that it already working for someone using nixos :)

**Comment by tfmoraes** [comment]
There is this other bug report https://github.com/NixOS/nixpkgs/issues/108995

**Comment by nathanlesage** [comment]
Marking this as external. @schlichtanders Would you mind trying out the most recent 2.0 beta and see if the bug occurs there as well? Might've been fixed in one of the electron versions since 1.8.9 …

**Comment by nathanlesage** [comment]
Closing due to inactivity. If the bug persists and any new info is available, feel free to open a PR, which I can then merge!
